<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件查看器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>页</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件处理</string>
    </property>
    <addaction name="actionLoadFile"/>
    <addaction name="actionUnloadFile"/>
    <addaction name="actionMergeFile"/>
    <addaction name="actionSplitFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>插件设置</string>
    </property>
    <addaction name="actionPluginMap"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dwFilePath">
   <property name="windowTitle">
    <string>文件列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>192</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
  </widget>
  <action name="actionLoadFile">
   <property name="text">
    <string>加载文件</string>
   </property>
  </action>
  <action name="actionMergeFile">
   <property name="text">
    <string>合并文件</string>
   </property>
  </action>
  <action name="actionSplitFile">
   <property name="text">
    <string>分割文件</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于文件封装器</string>
   </property>
  </action>
  <action name="actionPluginMap">
   <property name="text">
    <string>映射插件</string>
   </property>
  </action>
  <action name="actionUnloadFile">
   <property name="text">
    <string>卸载文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
